dsb-gui/src/main/java/com/spectralogic/dsbrowser/gui/components/localfiletreetable/LocalFileTreeTablePresenter.java

```diff
@@ -513,30 +508,36 @@ private void startPutJob(final Session session,
                              final String targetDir,
                              final String priority,
                              final JobInterruptionStore jobInterruptionStore,
-                             final TreeItem<Ds3TreeTableValue> treeItem) {
+                             final TreeItem<Ds3TreeTableValue> remoteDestination) {
         final Ds3PutJob putJob = new Ds3PutJob(session.getClient(), files, bucket, targetDir, priority,
                 settingsStore.getProcessSettings().getMaximumNumberOfParallelThreads(), jobInterruptionStore, deepStorageBrowserPresenter,
                 session, settingsStore, loggingService, resourceBundle);
         jobWorkers.execute(putJob);
         putJob.setOnSucceeded(event -> {
-            LOG.info("Succeed");
-            refreshBlackPearlSideItem(treeItem);
+            LOG.info("BULK_PUT job " + putJob.getJobId() + " Succeed.");
```

Comment:
This should use the logging statements formatted like: `LOG.info("BULK_PUT job  {} Succeed.", putJob.getJobId())`